### DISCUSSION TITLE
A beginner's doubt

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#2415] [General]
Hi everyone, I have a beginner's doubt that I didn't find in the documentation.

I'm using the liquidsoap 2.0.4 to stream audio with shoutcast/icecast. When I start the liquidsoap I can see on the logs, many infos about video, but I'm not using video, only audio like I said. This is normal? is posible to disable all about "video" maybe to "save resources" ?

```
2022/05/25 13:00:04 [clock:3] Using builtin (low-precision) implementation for latency control
2022/05/25 13:00:06 [frame:3] Using 44100Hz audio, 25Hz video, 44100Hz main.
2022/05/25 13:00:06 [frame:3] Video frame size set to: 1280x720
2022/05/25 13:00:06 [frame:3] Frame size must be a multiple of 1764 ticks = 1764 audio samples = 1 video samples.
2022/05/25 13:00:06 [frame:3] Targeting 'frame.duration': 0.04s = 1764 audio samples = 1764 ticks.
2022/05/25 13:00:06 [frame:3] Frames last 0.04s = 1764 audio samples = 1 video samples = 1764 ticks.
2022/05/25 13:00:06 [sandbox:3] Sandboxing disabled
2022/05/25 13:00:06 [video.converter:3] Using preferred video converter: ffmpeg.
2022/05/25 13:00:06 [audio.converter:3] Using samplerate converter: ffmpeg.

```

I'm only streaming .mp3 files inside playlists files to shoutcast/icecast servers.

Thanks, any help is appreciated.

**Comment by smimram** [comment]
Hi, this just prints the format of video that Liquidsoap would use if you had some, but no extra resources should be taken if you have only audio streams (no video is generated / encoded, etc.)
